.ci_support/linux_aarch64_target_platformlinux-aarch64.yaml

```diff
@@ -0,0 +1,55 @@
+BUILD:
+- aarch64-conda_cos7-linux-gnu
+c_compiler:
+- gcc
+c_compiler_version:
+- '7'
+cdt_arch:
+- aarch64
+cdt_name:
+- cos7
+channel_sources:
+- conda-forge,c4aarch64,defaults
+channel_targets:
+- conda-forge main
+docker_image:
+- condaforge/linux-anvil-aarch64
+expat:
+- '2.2'
+fontconfig:
+- '2.13'
+freetype:
+- 2.9.1
+icu:
+- '64.2'
+jpeg:
+- '9'
+libpng:
+- '1.6'
+libtiff:
+- 4.0.10
+libwebp:
+- 1.0.0
```

Comment:
@conda-forge/core it seems that only libwebp-1.0.2 has been rebuilt for for ppc64le, but 1.0.0 is pinned.

Reply:
See https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/316